lib/Backends/NNPI/tests/NNPIParameterSweepTest.cpp

```diff
@@ -18,3 +18,114 @@
 using namespace glow;
 
 std::set<std::string> glow::backendTestBlacklist = {};
+
+struct BlacklistInitializer {
```

Comment:
BlacklistInitializer is is a dummy struct just so initialzation would happen.
as I want to add tests through a loop(makes the file more readable), I can't initialize backendTestBlacklist outside a function, thus I do it inside the BlacklistInitializer c'tor.